docs/tutorials/pystac-spacenet-tutorial.ipynb

```diff
@@ -438,7 +433,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python [default]",
+   "display_name": "Python 3",
```

Comment:
This may or may not upset `nbsphinx` for the Read The Docs rendering after merge - if so, I'll set this back to [default]

Reply:
Nope, all good!